src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -47,6 +62,18 @@ export class DashboardComponent implements OnInit, OnDestroy {
 
     this.osdSettings$ = this.osdService.getOsdSettings();
     this.capacity$ = this.clusterService.getCapacity();
+    this.getHealth();
```

Comment:
```suggestion
```

We are not using the health data, so we don't need to use the healthService

Reply:
We are using the healthData to populate the health icon for now and eventually we will use it to show more info on why the health is in warning or error.